src/modal/modal-window.ts

```diff
@@ -76,8 +81,7 @@ export class NgbModalWindow implements OnInit,
     } else {
       elementToFocus = body;
     }
-    elementToFocus['focus'].apply(elementToFocus, []);
-
+    elementToFocus.focus();
```

Comment:
Do you know why we've called it through `apply` before and don't do this here any more?

Reply:
I have no clue at all why _you_ wrote all of these with `apply()`. The only thing I know is that it seems to be not needed anymore. It works fine without it. So it makes the code easier to read